CONTRIBUTING.md

```diff
@@ -53,6 +54,15 @@ It would be highly appreciated if contributions are backported to LTS branches i
 The list of maintained branches can be found in the [Current Branches section
 of BRANCHES.md](BRANCHES.md#current-branches).
 
+development branch
+------------------
+
+development branch is where all development happens. Some PRs will be backported into the LTS branches, but not all. So all PRs must first start with development. We won't merge a PR into development if we want backport PRs and they are not present. Very specifically, gatekeepers should merge all PRs at the same time. 
+
+Now, some people will create the backports at the same time as the main PR. If changes are required, this means extra work.
+
+It is fine to wait until we are happy with a PR before creating backports. But it's usually best to start with a PR against development (obviously there are special cases of issues that are only in an LTS branch, but ignoring that for the sake of a generally applicable rule) and then create backport PRs if required or requested. 
```

Comment:
We clearly need something, since at least one person didn't find what we currently have straightforward enough. However, I'm not sure that my rambling GitHub comments are the best fit for the documentation here: I'm very happy for others to suggest wording